h/migrations/versions/9e01b7287da2_remove_duplicates_from_user_group_table.py

```diff
@@ -0,0 +1,57 @@
+"""
+Remove duplicates rows from the user_group table.
+
+Revision ID: 9e01b7287da2
+Revises: 6f86796f64e0
+Create Date: 2016-07-08 17:54:57.399139
+"""
+
+from __future__ import unicode_literals
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+revision = '9e01b7287da2'
+down_revision = '6f86796f64e0'
+
+Base = declarative_base()
+Session = sessionmaker()
+
+
+user_group = sa.table('user_group',
+                      sa.Column('user_id', sa.Integer),
+                      sa.Column('group_id', sa.Integer))
+
+
+def upgrade():
+    session = Session(bind=op.get_bind())
+
+    # Find all the groups of duplicate user_group rows that have the same
+    # user_id and group_id values.
+    groups = (
+        session.query(user_group)
+        .group_by('user_id', 'group_id')
+        .having(sa.func.count('*') > 1))
+
+    for group in groups:
+        user_id = group[0]
+        group_id = group[1]
```

Comment:
It's a small thing, but it is simpler and clearer in situations like this to unpack the tuple in the `for` statement itself:

```python
    for user_id, group_id in groups:
        ...
```